lib/oli_web/live/new_course/course_details.ex

```diff
@@ -24,20 +24,30 @@ defmodule OliWeb.Delivery.NewCourse.CourseDetails do
         Ecto.Changeset.fetch_field(assigns.changeset, :class_days) |> elem(1)
       )
 
+    assigns =
+      assign(
+        assigns,
+        :class_modality,
+        Ecto.Changeset.fetch_field(assigns.changeset, :class_modality)
+        |> elem(1)
+      )
```

Comment:
Maybe here we could add the key and the value in the variable `assigns` that is declared above(line 20) as a keyword list or a map, WDYT?